_data/team/roles/relicensing_coordinator.yml

```diff
@@ -7,5 +7,5 @@ tasks:
   - Managing the switch to a new license
```

Comment:
Let's leave Relicensing Team up for now, at least until we're fully LGPLv2.1+. Eventually we may be able to retire it to history :-)

Reply:
Retirement will be very well deserved!!!